UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml

```diff
@@ -21,13 +21,12 @@
 			<skia:SKSwapChainPanel
 				x:Name="swapChain"
 				PaintSurface="OnPaintSwapChain"
-				PointerMoved="{x:Bind OnPointerMoved}"
-				Background="Transparent"
+				PointerMoved="OnPointerMoved" 
```

Comment:
Why did you remove the x:Bind ?

Reply:
Wasn't working. I raises method/event not found. 
Probably because the SkiaSharp package upgrade.

Reply:
Hummm! It was raising this:
System.Exception
  Message=Failed to load Page SkiaSharpTest.MainPage
 It was necessary to Droid for example, to make it works.

However, i tested the Bind now with the preview version (the one I used to make WASM work correctly and the Bind is working perfectly. Downgrading to 2.80.3 will make it not work. 

Will make a squash now, undoing the Bind changes.